TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
+        ".*" : {
+            "properties" : {
+                "byteOffset" : {
+                    "type" : "integer",
+                    "minimum" : 0
+                }
+            }
+        }
+    },
+    "definitions" : {
+        "featureReference" : {
+            "type" : "object",
+            "required" : ["byteOffset"]
+        },
+        "featureArray" : {
+            "type" : "array",
+            "items" : {
+                "type" : "number"
```

Comment:
> I can make it featureNumericArray if you'd like, but in general if you wanted to make a schema for a property that had something besides numbers in an array, you would need to make a new definition for that anyway.

That is OK, we just don't want this name to be too generic because it sounds like it is for all arrays.